### PR TITLE
Sessions/add session tracks

### PR DESCRIPTION
**Pull request by emb03**
@z3cka plz merge to stage, Gary wants this up for review ASAP

### CODE REVIEW

**Comment by z3cka** [comment]
reviewing now

**Comment by emb03** [comment]
Awesome

File: pages/03.program/program.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ process:
     markdown: true
     twig: true
 visible: true
+tepplate: sessions
```
Comment (z3cka) [inline pages/03.program/program.md:12]: hmmm, looks like a typo here... seems to not be affecting the site. **Suggest removing this line**.

Comment (emb03) [inline pages/03.program/program.md:12]: Will fix along with other client changes after review

**Pull request merged**